pages/docs/data-modeling/add-data-model.mdx

```diff
@@ -19,9 +19,9 @@ When possible, you should try to modify or extend an existing data model instead
 
 1. From your [project homepage](https://hashboard.com/app/), click `New Model`.
 2. Select the data connection you'd like to use or `Uploads` if you want to use [a file from your computer](/docs/data-modeling/query-data-files).
-3. Select either an existing table or click `SQL Editor` to use a custom query.
-4. Click `Build Model` to create the initial data model (all date fields and a row count measure are added by default).
+3. Select either an existing table or click `SQL editor` to use a custom query.
+4. Click `Continue` to create the initial data model (all date fields and a row count measure are added by default).
 5. Edit the name of the data model at the top of the page.
-6. Add additional [measures](/docs/data-modeling/measures) and [attributes](/docs/data-modeling/attributes) by clicking `Add measure` and `Add attribute` next to the name of the field.
-7. Create SQL-based custom measures and attributes  by clicking `+ Add custom measure` or `+ Add custom attribute`, respectively. See [Measures](/docs/data-modeling/measures) or [Attributes](/docs/data-modeling/attributes) for more details.
-8. Click `Save Model` then click `Open in Explore` to start exploring the data.
+6. Add additional [measures](/docs/data-modeling/measures) and [attributes](/docs/data-modeling/attributes) by clicking the `Add measure` and `Add attribute` icons next to the name of the field.
```

Comment:
Since the steps pane already has numbered instructions, it might be nice to walk them through those steps more explicitly

Reply:
Sure lemme draft another option

Reply:
Honestly, not sure there's an elegant way to do display two different sets of numbered instructions here. Any thoughts specifically on how?

Reply:
Yeah the concern here is that it would therefore look like this:

1. Choose data source
  1. .....
2. Build model
  1. .....
  2. .....
...

Which looks a little awkward maybe?